src/views/statusView.ts

```diff
@@ -406,19 +413,20 @@ export default class StatusView implements vscode.Disposable {
             return;
         }
         const self = this;
-        let index = 0;
-        let progressTicks = ["|", "/", "-", "\\"];
-
         let bar = this.getStatusBar(fileUri);
-        bar.progressTimerId = setInterval(() => {
-            index++;
-            if (index > 3) {
-                index = 0;
-            }
-
-            let progressTick = progressTicks[index];
-            statusBarItem.text = statusText + " " + progressTick;
+        let milliseconds = 0;
+        bar.queryTimer = setInterval(() => {
+            milliseconds += 1000;
+            //convert milliseconds to display time
+            const timeString = self.formatMilliseconds(milliseconds);
```

Comment:
Rename the method to `displayFormattedMilliseconds`, so that you can also get rid of the comment above this call.